Makefile.include

```diff
@@ -222,17 +222,21 @@ export PREFIX ?= $(if $(TARGET_ARCH),$(TARGET_ARCH)-)
 
 # Add standard include directories
 INCLUDES += -I$(RIOTBASE)/core/include -I$(RIOTBASE)/drivers/include -I$(RIOTBASE)/sys/include
-INCLUDES += -I$(RIOTCPU)/$(CPU)/include
+
+# Include Board configuration
 INCLUDES += -I$(RIOTBOARD)/$(BOARD)/include
+include $(RIOTBOARD)/$(BOARD)/Makefile.include
```

Comment:
There could be a reason to include defaultmodules.inc.mk and pseudomodules.inc.mk in the same order as before, in case they are used by the board and cpu makefiles.

Reply:
There are currently no difference, but I agree there is no reason to  change the order. I will fix it.